local.h

```diff
@@ -55,3 +55,4 @@ int local_read(struct local_socket *s);
 int local_header(struct local_socket *s);
 struct local_socket *local_socket_create(int fd);
 void local_socket_destroy(int i);
+const char * local_kind(int kind);
```

Comment:
Hi! This is very much minor nitpicking, but for consistency please attach the star to the function name, as in `*local_kind`.

Reply:
and there's a typo in your commit message `/conatined/contained/`

Reply:
Whups. ;) Chaned. I directly squashed, since this is a so tiny change. I hope this is okay.

Reply:
If u are interested why I am doing this: https://github.com/PolynomialDivision/babeld-ids/blob/main/src/main.c

This is just a small example application to show what u can do with it and how to programm.
At the end, c programming becomes very easy with ubus on OpenWrt. And it is very extensible and stable. I hope that more people will follow and write extensions and stuff like this. :) Mesh Routing is not dead! :)